tests/io/test_hf_datasets.py

```diff
@@ -18,8 +17,10 @@
 @pytest.mark.skipif(
     sysconfig.get_platform() == 'linux-aarch64', reason='libsndfile.so is missing on Linux ARM instances in CI'
 )
-@rerun(reruns=3, reruns_delay=15)  # Guard against connection errors downloading datasets
+# @rerun(reruns=3, reruns_delay=15)  # Guard against connection errors downloading datasets
```

Comment:
De-comment this or it will cause us grief in merge queue.

Btw you can do

`pytest --no-rerun`

if you're trying to run tests locally and want to disable reruns without commenting things out in the source

Reply:
Done.